source/adapters/level_zero/queue.cpp

```diff
@@ -24,8 +24,6 @@
 #include "ur_util.hpp"
 #include "ze_api.h"
```

Comment:
will this allow for all the `queue_legacy` to be reverted?

Reply:
Yes, I have a revert commit ready but it might be better to merge it separately: https://github.com/oneapi-src/unified-runtime/pull/1919